README.md

```diff
@@ -1 +1,42 @@
-# .github
\ No newline at end of file
+# About Us 
+
+[The Childhood Cancer Data Lab](https://www.ccdatalab.org/) was established by Alex’s Lemonade Stand Foundation (ALSF) in 2017. The Data Lab is a team of data scientists, designers, engineers, and communicators. Our mission is to accelerate the pace of finding novel cures and treatments for childhood cancer by putting resources and knowledge in the hands of pediatric cancer experts. 
+
+We construct tools that make vast amounts of data widely available, easily mineable, and broadly reusable. We also train researchers to better understand their own data and to advance their work more quickly. The Data Lab team simultaneously contributes to childhood cancer research and to the open science and open source software communities.
```

Comment:
One sentence per line helps with commenting, etc. when text is under version control. I don't expect that it changes how the rendered version looks (assuming it behaves like every other README on GitHub).

```suggestion
[The Childhood Cancer Data Lab](https://www.ccdatalab.org/) was established by Alex’s Lemonade Stand Foundation (ALSF) in 2017. 
The Data Lab is a team of data scientists, designers, engineers, and communicators. 
Our mission is to accelerate the pace of finding novel cures and treatments for childhood cancer by putting resources and knowledge in the hands of pediatric cancer experts. 

We construct tools that make vast amounts of data widely available, easily mineable, and broadly reusable. 
We also train researchers to better understand their own data and to advance their work more quickly. 
The Data Lab team simultaneously contributes to childhood cancer research and to the open science and open source software communities.
```